core/src/main/scala/com/nvidia/spark/rapids/tool/udf/EstimateEventRapidsUDF.scala

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2021-2023, NVIDIA CORPORATION.
```

Comment:
Nit: Update copyright year in this file and all other files where applicable.

Reply:
+1 it should be `Copyright (c) 2024` for newly added files